Makefile

```
@@ -48,8 +48,12 @@ new_connector:  # $ make new_connector type=Magento
 
 .PHONY: build
 build:
+	# /!\ Discovered that sometimes, this is missing !
+	pip3 install wheel
```

```suggestion
	python -m pip install wheel
```
depending on how python is distributed, `pip3` may not be available, or not point to the environment you want